internal/services/mysql/mysql_flexible_server_resource_test.go

```diff
@@ -361,14 +361,21 @@ func TestAccMySqlFlexibleServer_updateStorage(t *testing.T) {
 
 	data.ResourceTest(t, r, []acceptance.TestStep{
 		{
-			Config: r.updateStorage(data, 20, 360, true),
+			Config: r.updateStorage(data, 20, 360, true, false),
 			Check: acceptance.ComposeTestCheckFunc(
 				check.That(data.ResourceName).ExistsInAzure(r),
 			),
 		},
 		data.ImportStep("administrator_password"),
 		{
-			Config: r.updateStorage(data, 34, 402, false),
+			Config: r.updateStorage(data, 34, 402, false, false),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep("administrator_password"),
+		{
+			Config: r.updateStorage(data, 34, nil, false, true),
```

Comment:
You may want to just copy a test config where `iops` is absent

Reply:
I added a test config `updateStorageNoIOPS` where `iops` is not set and used this in the test `TestAccMySqlFlexibleServer_updateStorage`